packages/bridge-ui-v2/src/components/Bridge/NFTBridgeSteps/ConfirmationStep.svelte

```diff
@@ -143,8 +143,21 @@
     }
   }
 
+  const _notAllowBridge = ()  => {
+    const conditions = [
+      !$bridgeService,
+      !$selectedToken,
+      !$network,
+      !$destNetwork?.id,
+      !$account?.address,
+    ]
+
+    return conditions.some(Boolean)
+  }
```

Comment:
```suggestion
  const canBridge = ()  => {
    const conditions = [
      $bridgeService,
      $selectedToken,
      $network,
      $destNetwork?.id,
      $account?.address,
    ]
    return conditions.some(Boolean)
  }
```

as per your own standard from a previous PR, it would be better to use the positive check.

Reply:
@KorbinianK I think it must be 'every', why 'some'?

Reply:
Yes absolutely, for the inverted check it should be `every`.